source/_components/ads.markdown

```diff
@@ -102,7 +102,7 @@ name:
   required: false
   type: string
 device_class:
-  description: The [type/class](/components/binary_sensor/) of the sensor to set the icon in the frontend.
+  description: The [type/class](https://developers.home-assistant.io/docs/en/entity_binary_sensor.html#available-device-classes) of the sensor to set the icon in the frontend.
```

Comment:
Linking to the developer docs, for a user configuration, just seems very wrong. This is either a user feature documented in the user docs, or a developer feature...

Reply:
Or both userdocs and developer feature.

I agree links from user docs to dev docs should be forbidden unless the explicit need it to redirect developers to the right docs (which is absolutely not the case here).

Nevertheless, it solves an (apparently) direct need from users for now.

So I'm fine with it for now. Searching for "developers.home-assistant.io" in our documentation can help us solve / fix things better later on.

Reply:
So I agree with both of you. I think we did jump the gun by removing the device classes, and there are more use cases than I anticipated, and linking to the dev docs is not good practice.

Let's merge this, and then let's do a new PR to put the device classes back

Reply:
I did a new commit where the old links are back, with a small change in the description everywhere because it not only changes the icon but also the state.